docker-compose.yml

```diff
@@ -4,28 +4,28 @@ services:
   ####################### Keycloak #######################
   keycloak:
     restart: on-failure:1
-    container_name: pims-keycloak
+    container_name: psp-keycloak
```

Comment:
I like this change so it can co-exist locally with CITZ pims. 
Note that you would need to also change the volume names at the end of this file so they don't override each other (PSP and CITZ pims)

GitHub is not letting me add a comment on the specific lines but I mean these:

```yaml
####################### Volumes Definition #######################
volumes:
  api-db-data:
    name: pims-api-db-data  <--- need to be changed to "psp-api-db-data" or similar
  frontend-node-cache:
    name: pims-frontend-node-cache

```

Reply:
Yes I have a PR in CITZ PIMS for this.

Reply:
well we should change this here to PSP right?

Reply:
Like CITZ pims can keep the original volume names but maybe change them for PSP?

Reply:
Otherwise you could end up using CITZ pims database and node_modules in a PSP local docker deployment

Reply:
No, I submitted a PR in CITZ to rename things.  It impacts more people if I change it here for MOTI.  The change I've submitted here will allow user preferences for ports and will allow you all to keep you db.